support-files/mariadb.service

```diff
@@ -0,0 +1,104 @@
+#
+#  /etc/systemd/system/mariadb.service
+#
+
+[Unit]
+Description=MariaDB database server
+After=network.target
+After=syslog.target
+Requires=%p.socket
+
+[Install]
+WantedBy=multi-user.target
+Alias=mysql.service
+Alias=mysqld.service
+
+
+[Service]
+
+##############################################################################
+## Core requirements
+##
+
+Type=Notify
+
+# Setting this to true can break replication and the Type=Notify settings
+PrivateNetwork=false
+
+##############################################################################
+## Package maintainers
+##
+
+User=mysql
+
+# Execute pre and post scripts as root
+PermissionsStartOnly=true
+
+# Needed to create system tables etc.
+ExecStartPre=/usr/bin/mariadb-systemd-start
+
+# Start main service
+# skip-networking as its socket activation based
+ExecStart=/usr/sbin/mysqld $OPTIONS
```

Comment:
Okay, but this needs to be documented. For starters, could you please add a comment about it? here, just above line 42. thanks.